src/Init/Core.lean

```diff
@@ -1680,40 +1680,64 @@ So, you are mainly losing the capability of type checking your development using
 -/
 axiom ofReduceNat (a b : Nat) (h : reduceNat a = b) : a = b
 
+end Lean
+
```

Comment:
I just noticed that the classes were moved out of the `Lean` namespace. This is good, it's a confusing namespace for these algebraic classes. However, the root namespace is perhaps not the right place either. It causes annoyances to anyone who wants to use one of these names in their namespace and these are pretty common names. Root is better than `Lean` but any meaningful namespace is better than either.

Reply:
Good point.  I'll put them in `Std`.